Makefile

```diff
@@ -324,6 +324,25 @@ define stringreplace
 	$(build_depsbindir)/stringreplace $$(strings -t x - $1 | grep '$2' | awk '{print $$1;}') '$3' 255 "$(call cygpath_w,$1)"
 endef
 
+JL_HEADERS := \
```

Comment:
My prior feedback is now collapsed, but I still don't like this form where `JL_HEADERS` needs to be kept up to date. A shorter exclusions list of things that are currently included in `src/support/*.h` but aren't needed would be more maintainable.